src/components/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Typography from './Typography';
+import { Grid } from '@mui/material';
+
+export default {
+  title: 'Components/Typography',
+  component: Typography,
+  argTypes: {
+    children: {
+      control: {
+        type: 'element',
+        require: true,
+      },
+    },
+    variant: {
+      control: {
+        options: [
+          'h1',
+          'h2',
+          'h3',
+          'h4',
+          'h5',
+          'h6',
+          'subtitle1',
+          'subtitle2',
+          'body1',
+          'body2',
+          'caption',
+          'button',
+          'overline',
+          'inherit',
+        ],
+        type: 'radio',
+      },
+    },
+    align: {
+      control: {
+        type: 'radio',
+      },
+    },
+    gutterBottom: {
+      control: { type: 'boolean' },
+    },
+    noWrap: {
+      control: { type: 'boolean' },
+    },
+    paragraph: {
+      control: { type: 'boolean' },
+    },
+    fontFamily: {
+      control: {
+        options: ['DM Mono', 'Arizona Serif', 'default'],
+        type: 'radio',
+      },
+    },
+  },
+} as ComponentMeta<typeof Typography>;
+
+const Template: ComponentStory<typeof Typography> = (args) => (
+  <Typography {...args} />
+);
+
+const MultipleTemplates: ComponentStory<typeof Typography> = (args) => (
+  <>
+    <Typography {...args} />
+    <Typography {...args} />
```

Comment:
What is this for? Why duplicate the text?

Reply:
Used it for `gutterBottom`, to show that it adds margin-bottom to text. I add it just to display that it works and how it looks

Reply:
Right, otherwise you can't see the empty space.